protocol/src/common/protocol.callHierarchy.ts

```diff
@@ -59,6 +59,7 @@ export namespace CallHierarchyPrepareRequest {
 	export const messageDirection: MessageDirection = MessageDirection.clientToServer;
 	export const type = new ProtocolRequestType<CallHierarchyPrepareParams, CallHierarchyItem[] | null, never, void, CallHierarchyRegistrationOptions>(method);
 	export type HandlerSignature = RequestHandler<CallHierarchyPrepareParams, CallHierarchyItem[] | null, void>;
+	export const capabilities: { client: 'textDocument.callHierarchy'; server: 'callHierarchyProvider' } = { client: 'textDocument.callHierarchy', server: 'callHierarchyProvider' };
```

Comment:
@dbaeumer Do you think we should add capability information for requests like `callHierarchy/incomingCalls` that don't define their own capabilities and instead depend on whether the server supports other methods (in this case `textDocument/prepareCallHierarchy`)?